server/src/users/users.controller.ts

```diff
@@ -18,7 +18,7 @@ export class UsersController {
   }
 
   @Get(':id')
-  findOne(@Param('id') id: string) {
+  async findOne(@Param('id') id: string) {
```

Comment:
#### Fix 
Ignore if id is in fact a string

For all the controller methods using an id that should be an int use the `ParseIntPipe`, and follow the example in [NestJs](https://docs.nestjs.com/pipes)